### ISSUE TITLE
End & Nether Biomes not spawning

### ISSUE THREAD

**Issue by toothmonsterr** [NillerMedDild/Enigmatica6#1075]
**World made in 0.3.6, currently running 0.3.8**

**Describe the bug**
Not sure why, but I haven't been able to get End biomes from BYG or Biomes'o Plenty to spawn. I believe the Nether is rendering as vanilla as well. The server.properties has the level-type as default as advised by the discord.

**Comment by toothmonsterr** [comment]
Nature's Compass returns Not Found for any modded Nether or End biome in respective dimensions.

**Comment by NillerMedDild** [comment]
What is your world type set to? :)

**Comment by toothmonsterr** [comment]
It's set to default.

> The server.properties has the level-type as default as advised by the discord.

The overworld is generating biomes from both mods just fine.

**Comment by NillerMedDild** [comment]
Oh right, server, sorry ^^'

Did the Ender Dragon spawn in The End?

**Comment by toothmonsterr** [comment]
It did, yeah, and has been killed.

**Comment by NillerMedDild** [comment]
Would you mind sending me your worlds `level.dat` file?

**Comment by toothmonsterr** [comment]
Sure. Here it is on [dropbox](https://www.dropbox.com/s/3oolthzdqw4cj39/level.dat?dl=0).

**Comment by NillerMedDild** [comment]
For whatever reason BYG isn't enabled in The Nether or End for you, I'll fix it real quick. If you want the dimensions to regen from scratch delete the dimension folders :)

**Comment by toothmonsterr** [comment]
Thank you!

**Comment by toothmonsterr** [comment]
Would purging chunks work instead? I don't want to delete progress and builds my players have made there.

**Comment by NillerMedDild** [comment]
yeah it should. or you can just generate new terrain, that should work too

**Comment by NillerMedDild** [comment]
Here you go: https://drive.google.com/file/d/1Mli6mxuGK0ChVbeWU76VeR3spbDz5ksL/view?usp=sharing
Let me know when you've downloaded it so I can take it down :)

Remember to make a backup before swapping the level.dat files

**Comment by toothmonsterr** [comment]
Got it, thanks so much!

**Comment by toothmonsterr** [comment]
Okay, maybe closed the issue a bit prematurely...

After replacing the level.dat, the biomes still aren't generating, and Nature's Compass still shows not found.

**Comment by NillerMedDild** [comment]
Damn :P you're certain it's new chunks?

**Comment by toothmonsterr** [comment]
Yup, TP'd myself to the middle of nowhere to test in both dimensions.

**Comment by NillerMedDild** [comment]
Dammit. May I have your "new" level.dat? :P

**Comment by toothmonsterr** [comment]
[Here you go.](https://www.dropbox.com/s/v2w38g1wkrlpwfz/level.dat?dl=0)

**Comment by NillerMedDild** [comment]
Did you make the level.dat change while the server was still running? That one is the same as the first.

**Comment by toothmonsterr** [comment]
Yeah, I did, that makes sense. Let me try again.

**Comment by toothmonsterr** [comment]
The End biomes are generating now but my Nether has ceased to exist. I cannot use a portal to it, and I cannot tp to it either.

**Comment by toothmonsterr** [comment]
This is the error from my server console, not sure if it's related to why:
`[Server] [main/ERROR] [minecraft/SaveFormat]: WorldGenSettings: Input does not contain a key [type]: MapLike[{seed:-731776721379764655L,preset:"byg:bygnether"}] missed input: {"minecraft:the_nether":{generator:{settings:"minecraft:nether",seed:-731776721379764655L,biome_source:{seed:-731776721379764655L,preset:"byg:bygnether"},type:"minecraft:noise"},type:"minecraft:the_nether"}}`

**Comment by NillerMedDild** [comment]
Found my mistake :/ new file: [level.dat](https://drive.google.com/file/d/1K_3MBc8JDrfI6P_a501SuUVhziFIsW7B/view?usp=sharing)

**Comment by toothmonsterr** [comment]
Sorry for the delay, I was in class.
Now the nether is generating BYG biomes but the End is inaccessible in the same way as before with the Nether.
[level.dat again](https://www.dropbox.com/s/32b4f6z3kylp9bd/level.dat?dl=0)

**Comment by toothmonsterr** [comment]
Should note this is after updating to 3.10.

**Comment by toothmonsterr** [comment]
Fixed by editing with NBTEditor.

**Comment by NillerMedDild** [comment]
For future reference, what I did was edit his `level.dat` file (found in the world/save) to match this image:
https://user-images.githubusercontent.com/21295394/105964615-be521f00-6082-11eb-8385-202bc3a9246d.png

But using the seeds already found in his `level.dat`

**Comment by LivingSkeleton11** [comment]
I've been having the same problem, I tried uninstalling all other biome-related mods I had (Nullscape, Terralith, Amplified Nether) and leaving only BYG and TerraBlender (a mod BYG needs to run on 1.18). I tried following all the steps you laid out, to no avail. I even made new worlds with just BYG/TerraBlender, and the biomes spawn just fine in the overworld, but nothing in the Nether or End. They don't even appear as options in /locatebiome.
